test/tf/gen_tf_pb.py

```diff
@@ -120,19 +120,21 @@ def batchnorm_test(g1):
     with g1.as_default():
         g1_input = tf.compat.v1.placeholder(tf.float32,
                                             shape=(1, 16, 16, 32),
-                                            name='0')
-        g1_scale = tf.constant(1.0, dtype=tf.float32, shape=[32], name='1')
-        g1_offset = tf.compat.v1.placeholder(tf.float32, shape=(32), name='2')
-        g1_mean = tf.compat.v1.placeholder(tf.float32, shape=(32), name='3')
+                                            name='x')
+        g1_scale = tf.constant(1.0, dtype=tf.float32, shape=[32], name='scale')
```

Comment:
Would be good to have `fp16` test and a test that would hit `unsqueeze` lines for batch norm parameters.

Reply:
Not sure I understand what you meant by hitting the `unsqueeze` lines.

Reply:
Never mind about `unsqueeze`.  But fp16, test is good to have.